lib/addons/amp/index.ts

```diff
@@ -60,7 +48,7 @@ export const enum DeploymentMode {
  * Defaults options for the add-on
  */
 const defaultProps = {
-    workspaceName: 'blueprints-amp-workspace',
+    ampPrometheusEndpoint: "https://aps-workspaces.us-east-1.amazonaws.com/workspaces/ws-sample/",
```

Comment:
why is this endpoint here hardcoded?

Reply:
This is to show a sample. See the end it has `ws-sample`.

Reply:
remove it please from default values.

Reply:
We cant remove that parameter. `ampPrometheusEndpoint` is a mandatory parameter.